Tests/Unit/Controller/TeaControllerTest.php

```diff
@@ -51,7 +51,7 @@ protected function setUp(): void
         $this->viewMock = $this->createMock(TemplateView::class);
         $this->subject->_set('view', $this->viewMock);
 
-        $responseMock = $this->createMock(HtmlResponse::class);
+        $responseMock = $this->createStub(HtmlResponse::class);
```

Comment:
Shouldn't we also rename the variable from Mock to Stub?

Reply:
Good catch! Done.